src/packages/index.ts

```diff
@@ -1,15 +1,17 @@
 import { BlockstackConfigurationService } from "./configuration-service";
 import * as encryption from "./encryption";
-import * as Errors from "./errors";
+import * as error from "./error/index";
```

Comment:
- [ ] can use the syntax `import * form error from "./error";`

- [ ] please rename the package to `errors` to avoid conflicts with the standard error catching in `catch`